madgraph.sh

```diff
@@ -0,0 +1,28 @@
+package: madgraph
+version: "%(tag_basename)s"
+tag: "v2.8.1"
+source: https://github.com/mfasDa/MadGraph5
+---
+#!/bin/bash -e
+
+rsync -a --exclude='**/.git' --delete --delete-excluded $SOURCEDIR/ $INSTALLROOT/
+
+# Modulefile
+MODULEDIR="$INSTALLROOT/etc/modulefiles"
+MODULEFILE="$MODULEDIR/$PKGNAME"
+mkdir -p "$MODULEDIR"
+cat > "$MODULEFILE" <<EoF
```

Comment:
Can you use the (somewhat) recently introduced `alibuild-recipe-tools.sh` to generate the modulefile?

Reply:
OK, is there an example how to use it?

Reply:
@mfasDa 
you can check an example in `delpheso2.sh` recipe.

Reply:
@preghenella Yes, I have seen it. It is done.